app/models/ability.rb

```diff
@@ -106,7 +106,7 @@ def appointment_slot_abilities
 
   def beneficiary_abilities
     can :create, Beneficiary
-    can [:index, :show, :update], Beneficiary, created_by_id: @user_id
+    can %i[index show update delete], Beneficiary, created_by_id: @user_id
```

Comment:
can you add delete to the lines below so the order submitter and the order creator can also delete the beneficiary